project/src/main/webapp/catalog.js

```diff
@@ -1,75 +1,70 @@
 var photoURLS; 
-var image;
+const image = document.getElementsByClassName("carousel-image");
+const reviewArea = document.getElementById("review-space");
 var reviewArr;
-var reviewSpace;
-
 
 function showCatalog(placeID) {
     console.log ("this is the catalog")
     //returns photos associated with user search
-        const request = {
-            placeId: placeID,
-            fields: ["photos", "rating", "reviews"]
-        };
+    removeAllChildNodes(reviewArea);
+    containerThree = document.getElementById("container-3").removeAttribute("hidden");
+    messageContainer = document.getElementById("chat-div").removeAttribute("hidden");
+    const request = {
+        placeId: placeID,
+        fields: ["photos", "rating", "reviews", "name", "formatted_address"]
```

Comment:
why don't you delcare it outside this funciton scope if this is not going to change?

Reply:
request variable is used in other js files so it would cause an error if i declare outside of the function